oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/index/ElasticIndexEditorContext.java

```diff
@@ -61,7 +62,9 @@ public void enableReindexMode() {
         // Now, that index definition _might_ have been migrated by super call, it would be ok to
         // get writer and provision index settings and mappings
         try {
-            getWriter().provisionIndex();
+            long seed = UUID.randomUUID().getMostSignificantBits();
+            definitionBuilder.setProperty(ElasticIndexDefinition.PROP_INDEX_NAME_SEED, seed);
```

Comment:
question: is this automatically saved or some commit is needed?

Reply:
Gets saved automatically at a later stage in the indexing process.

Reply:
please add a relevant comment